### PR TITLE
Taking over egui_glium

### PR DESCRIPTION
**Pull request by fayalalebrun**
Hello. I would like to maintain egui_glium. I have already updated my fork to the newest version of glium. You can find it at: https://github.com/fayalalebrun/egui_glium

Let me know about next steps regarding access to crates.io.

### CODE REVIEW

**Comment by emilk** [comment]
This is great news, thank you!

Please make sure you remove all mentions of `egui_glium`, except for in `README.md` where it should be listed under the "3rd party integrations" heading.

I'll transfer ownership of the crate to you once this PR is merged - what is your crates.io handle?

**Comment by fayalalebrun** [comment]
Done. I am using this account for crates.io, so I believe the handle should be the same.

File: README.md [hunk 1]
```diff
@@ -227,6 +226,7 @@ These are the official egui integrations:
 * [`egui_skia`](https://github.com/lucasmerlin/egui_skia) for [skia](https://github.com/rust-skia/rust-skia/tree/master/skia-safe).
 * [`smithay-egui`](https://github.com/Smithay/smithay-egui) for [smithay](https://github.com/Smithay/smithay/).
 * [`tauri-egui`](https://github.com/tauri-apps/tauri-egui) for [tauri](https://github.com/tauri-apps/tauri).
+* [`egui_glium`](https://github.com/fayalalebrun/egui_glium) for compiling native apps with [Glium](https://github.com/glium/glium)
```
Comment (emilk) [inline README.md:229]: I keep the list sorted by the name of what we integrate with, so this should come between  `egui_glfw_gl` and `egui-glutin-gl`

**Comment by emilk** [comment]
Great - thank you! An invite has been sent on crates.io

**Pull request merged**